optic.yml

```diff
@@ -0,0 +1,55 @@
+capture:
+  openapi.yml:
+    server:
+      # The command to run your server.
+      # Optional: If omitted, Optic assumes the server is running or started elsewhere.
+      # command: your-server-command
+      # The url where your server can be reached once running.
+      # Required: Can be overridden with '--server-override'.
+
+      # QUESTION: Is it ok that both this value and proxy_variable are the api gateway url?
+      url: https://rv42fyt9b0.execute-api.us-east-1.amazonaws.com/try-optic
+      # A readiness endpoint for Optic to validate before sending requests.
+      # Optional: If omitted, perform no readiness checking.
+
+      ready_endpoint: /
+      # The interval to check 'ready_endpoint', in ms.
+      # Optional: default: 1000
+      ready_interval: 1000
+      # The length of time in ms to wait for a successful ready check to occur.
+      # Optional: default: 10_000, 10 seconds
+      ready_timeout: 10_000
+    # At least one of 'requests.run' or 'requests.send' is required below.
+    requests:
+      # Run a command to generate traffic. Requests should be sent to the Optic proxy, the address of which is injected
+      # into 'run.command's env as OPTIC_PROXY or the value of 'run.proxy_variable', if set.
+      run:
+        # The command that will generate traffic to the Optic proxy. Globbing with '*' is supported.
+        # Required if specifying 'requests.run'.
+        command: your-test-command
+        # The name of the environment variable injected into the env of the command that contains the address of the Optic proxy.
+        # Optional: default: OPTIC_PROXY
+
+        # QUESTION can we get this from environment variables, particularly the .env file
+        proxy_variable: https://rv42fyt9b0.execute-api.us-east-1.amazonaws.com/try-optic
```

Comment:
This should be named the environment variable that your requests run. Optic will inject the proxy variable where the server is running (optic usually starts on port 8000, but will find the open available port)

From looking at here https://github.com/muratkeremozcan/prod-ready-serverless/blob/main/__tests__/steps/when.js#L71 I think the correct thing to add here is `proxy_variable: rest_api_url`

Reply:
All right, so it is already the environment variable. Beautifu.